htdocs/core/lib/functions2.lib.php

```diff
@@ -1280,7 +1280,7 @@ function get_next_value($db, $mask, $table, $field, $where = '', $objsoc = '', $
 	$counter = 0;
 	$sql = "SELECT MAX(".$sqlstring.") as val";
 	$sql .= " FROM ".MAIN_DB_PREFIX.$table;
-	$sql .= " WHERE ".$field." LIKE '".$db->escape($maskLike)."'";
+	$sql .= " WHERE ".$field." LIKE '".$db->escape($maskLike)."%'";
```

Comment:
If user change its numbering rule, for example adding this will inteoduce a bug because search may find string that should not match when exact search is done.
So i recommand to add a constant SEARCH_FOR_NEXT_VAL_ON_START_ONLY to add this.
Also this is not a bug into v15, it is an enhancement to allow external module needs so thus must be pushed into develop.